rasa_nlu/extractors/__init__.py

```diff
@@ -25,3 +29,14 @@ def add_processor_name(self, entity):
         else:
             entity["processors"] = [self.name]
         return entity
+
+    def filter_trainable_entities(self, entity_examples):
+        # type: (List[Message]) -> TrainingData
+        for message in entity_examples:
+            entities = []
+            for ent in message.get("entities", []):
+                extractor = ent.get("extractor")
+                if not extractor or extractor == self.name:
+                    entities.append(ent)
+            message.set("entities", entities)
```

Comment:
I don't think this will work: This will remove all entities from the message that do not belong to the crf extractor, which is fine. The issue is, that this is modifying the message object directly, which will lead to the entities being removed for every component that comes after the crf component. So if there is any other component that needs some entity annotations after the crf, the entities will be missing.

Reply:
good point 👍